gpii/node_modules/flowManager/src/FlowManagerRequests.js

```diff
@@ -29,10 +29,25 @@
 
     require("./PrivacyFilter.js");
 
+    gpii.logFully = function () {
+        var oldRenderChars = fluid.logObjectRenderChars;
+        fluid.logObjectRenderChars = 2 << 20;
+        fluid.log.apply(null, arguments);
+        fluid.logObjectRenderChars = oldRenderChars;
+    };
+
+    gpii.renderMegapayload = function (megapayload) {
```

Comment:
I added a comment, since the function may well do more/different to this in future